sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/ZPageHttpHandler.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.extensions.zpages;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableMap;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
```

Comment:
I'm pretty hesitant to depend on anything in the `com.sun` namespace. There is no guarentee that all JDKs will support this, or that these classes will be around in the future.

Reply:
Is there any suggested alternative of a HttpServer? OpenCensus also used this way of implementation and seems like major JDKs supports this

Reply:
I'm not saying we can't use it..it just gives me pause. If we go ahead with this, we should make the documentation super clear about the dependency on a JRE/JDK that has these classes included.

Reply:
I spent some time researching this, and after consulting with some long-time JVM/JDK gurus (well, one guru: @kittylyst), I think this is fine. We should still document it, but I think this will be just fine, especially as a extension module.

Reply:
I see the dependency in build.gradle, I guess either way we're requiring it as a dependency, not relying on the JDK? Want to make sure I didn't miss something

Reply:
I included the dependency so that VSCode didn't give me any errors (it's complaining about the same issue as com.sun is not standard in all JDKs), does that help with the issue with JDK as well?

Reply:
yeah, interesting that this is also published to maven central. Pretty old datestamp on that release, but it does look to be the same code built-in to JDK 11, at least with a cursory examination of the classes.